app/public/electron.js

```diff
@@ -337,7 +337,7 @@ app.on('ready', () => {
   tray = new Tray(
     path.join(
       resourcesPath(), 'icons',
-      selectByOS({ mac: 'kopia-tray.png', win: 'kopia-tray.ico', linux: 'kopia-tray.png' })));
+      selectByOS({ mac: 'kopiaTrayTemplate.png', win: 'kopia-tray.ico', linux: 'kopia-tray.png' })));
```

Comment:
@rsammelson mainly curious, was it necessary to rename this file? why was that?

Reply:
The "Template" at the end of the filename is how it knows it should render as a template (separate light and dark versions) in electron.

I don't really think that is a great system but I guess it works.

Reply:
Ah! got it. Thanks for the explanation.